appcues/src/main/java/com/appcues/data/remote/DataLogcues.kt

```diff
@@ -0,0 +1,168 @@
+package com.appcues.data.remote
+
+import com.appcues.logging.Logcues
+import okhttp3.Headers
+import okhttp3.MediaType
+import okhttp3.Request
+import okhttp3.Response
+import okhttp3.internal.http.promisesBody
+import okio.Buffer
+
+internal class DataLogcues(private val logcues: Logcues) {
```

Comment:
yea I'm not sure what to name it, initially i was going for HttpLogcues, as its basically a Logcues wrapper that knows how to parse request/responses to the debug(message) methods.
I later on renamed to DataLocues because I wanted to use it in Repository (but maybe in the future we. could remove some logs from the repository calls since they may be repeating logs)

Open to ideas here for naming, I can add top level comments meanwhile.